src/logging/BaseLogger.ts

```diff
@@ -15,9 +15,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import {LogItem, LabelOrValues, FilterCreator, LogCallback} from "./LogItem";
-import {LogLevel, LogFilter, LogLevelOrNull} from "./LogFilter";
-import {Platform} from "../platform/web/Platform.js";
+import {LogItem} from "./LogItem";
+import {LogLevel, LogFilter} from "./LogFilter";
+import type {FilterCreator, LabelOrValues, LogCallback} from "./LogItem";
+import type {LogLevelOrNull} from "./LogFilter";
+// todo: should this import be here just for getting the type? should it instead be done when Platform.js --> Platform.ts?
```

Comment:
It's best to do it now already, even if the imported file is still javascript. That way, the conversion of this file is complete already, even though the types used here aren't fully typescripted.

Reply:
Thanks 👍, I figured as much and have done the same elsewhere.